cmd/kubernetes_test.go

```diff
@@ -0,0 +1,39 @@
+package cmd
+
+import (
+	"reflect"
+	"testing"
+)
+
+type InstallApplicationsArgs struct {
+	defaultApps []string
+	apps        string
+	removeApps  string
+}
+
+func TestInstallApplications(t *testing.T) {
+	var tests = []struct {
+		name string
+		args InstallApplicationsArgs
+		want []string
+	}{
+		{
+			name: "Test Removal of Default Apps",
+			args: InstallApplicationsArgs{defaultApps: []string{"metrics-server", "traefik"}, apps: "", removeApps: "traefik"},
+			want: []string{"metrics-server"},
+		},
+		{
+			name: "Test Installation of Default Apps and other Apps",
+			args: InstallApplicationsArgs{defaultApps: []string{"metrics-server", "traefik"}, apps: "foo", removeApps: ""},
+			want: []string{"metrics-server", "traefik", "foo"},
+		},
```

Comment:
Test case for :

 `InstallApplicationsArgs{defaultApps: []string{"metrics-server", "traefik"}, apps: "foo", removeApps: "traefik"}`